src/js/core/options.js

```diff
@@ -3,6 +3,7 @@
 const elBody = document.querySelector('body');
 const elNoBookmarks = document.getElementById('no-bookmarks');
 const elWhitelistTable = document.getElementById('whitelist-table');
+const elRemoveAllWrapper = document.getElementById('whitelist-remove-all-wrapper');
```

Comment:
nit: Could you move this one line up so that the DOM node variables are sorted alphabetically?

Reply:
Reordered